tests/sampling/test_forward.py

```diff
@@ -862,11 +862,18 @@ def test_logging_sampled_basic_rvs_prior(self, caplog):
         assert caplog.record_tuples == [("pymc.sampling.forward", logging.INFO, "Sampling: [x, z]")]
         caplog.clear()
 
+        # RV only
         with m:
             pm.sample_prior_predictive(draws=1, var_names=["x"])
         assert caplog.record_tuples == [("pymc.sampling.forward", logging.INFO, "Sampling: [x]")]
         caplog.clear()
 
+        # observed only
```

Comment:
Nitpick, the distinction is not RV vs observed, is free_RV vs observed_RV, but either way I don't think the comment helps.